assets/javascripts/discourse/templates/admin-wizards-transfer.hbs

```diff
@@ -0,0 +1,34 @@
+<h3>Select Wizards to export</h3>
```

Comment:
In terms of the html structure, try to find another admin panel within Discourse with a similar structure to base yours off. This pattern-based approach isn't always appropriate, but I think in this case we can make the UI a bit more similar to some existing Discourse admin UIs to make this feel a bit more native

For example take a look at the "Customize > Emoji" UI where you have a simple form-like structure. From that UI I would copy certain elements of the structure, for example the "admin-container" div and the us of ``h2`` tags.

Let's also add in a heading for the "Import" form (and change the headings to just "Export" and "Import")

<img width="557" alt="Screenshot at Jul 30 11-10-44" src="https://user-images.githubusercontent.com/5931623/62092962-c3ca7f80-b2ba-11e9-80e9-e579a35b0a18.png">